src/toil/jobStores/fileJobStore.py

```diff
@@ -87,17 +87,25 @@ def destroy(self):
     # existence of jobs
     ##########################################
 
-    def create(self, jobNode):
+    def getJobGraph(self, jobNode):
         # The absolute path to the job directory.
         absJobDir = tempfile.mkdtemp(prefix="job", dir=self._getTempSharedDir())
         # Sub directory to put temporary files associated with the job in
         os.mkdir(os.path.join(absJobDir, "g"))
         # Make the job
         job = JobGraph.fromJobNode(jobNode, jobStoreID=self._getRelativePath(absJobDir),
                                    tryCount=self._defaultTryCount())
-        # Write job file to disk
-        self.update(job)
         return job
+        
+    def create(self, jobNode):
+        jobGraph = self.getJobGraph(jobNode)
+        # Write job file to disk
+        self.update(jobGraph)
+        return jobGraph
+
+    def batchCreate(self, jobGraphs):
```

Comment:
Nitpick: I think it may be better to put the various "not actually batching" batchCreates as a default batchCreate method in abstractJobStore which just loops over the jobs and calls `self.create`. That way we could get rid of the other 4 batchCreates that don't really do multiple writes at once.